src/Bladeburner/Bladeburner.ts

```diff
@@ -135,7 +144,7 @@ export class Bladeburner {
     if (!availability.available) {
       return { message: `Could not start action ${action.name}: ${availability.error}` };
     }
-    this.action = actionId;
+    this.action = action.id;
```

Comment:
Why does your PR keep growing in scope? :(

This seems like a very subtle change, and it is not immediately obvious how it's related to everything else.

Reply:
The two actions are not of the same type.

Reply:
I still don't get it. `this.action` is an `ActionIdentifier | null`. There is no type advantage to assigning it a narrower type.

Reply:
Leftover from the previous iteration. No longer necessary. ty